### DISCUSSION TITLE
The supplied auth credential is malformed or has expired! Facebook Login

### DISCUSSION THREAD

**Discussion by Shajeel-Afzal** [firebase/flutterfire#10091] [Q&A]
I am getting an error that is not allowing Facebook login and giving the error as shown in the screenshot below:

![image](https://user-images.githubusercontent.com/3718283/206833382-258812a4-7a61-4fbe-8b6d-7308a01e92de.png)

I have configured the Facebook Authentication Properly on Firebase Console by providing the App ID and Secret. 

And in my Flutter code, I have specified the app id for the clientId parameter:

![image](https://user-images.githubusercontent.com/3718283/206834993-0381d274-de1d-4c37-85c5-7c5a5a9c2c8e.png)

I am using firebase_ui_oauth_facebook: ^1.0.8 right now.

**Comment by AdamBCo** [comment]
@Shajeel-Afzal Were you able to resolve this issue? What was the root cause?

**Reply by Shajeel-Afzal** [reply]
No, I could not solve this problem, so I skipped Facebook login completely from the app

**Reply by jsg2021** [reply]
Turning this off fixed it for me:
<img width="1053" alt="Screenshot 2024-03-29 at 9 10 24 PM" src="https://github.com/firebase/flutterfire/assets/577453/ea0f83eb-a1a7-4760-aa89-a188ccaadf78">

**Reply by michaelpomogajko** [reply]
@jsg2021 Where can I find this setting?
